Detectors/TRD/base/src/FeeParam.cxx

```diff
@@ -292,8 +292,8 @@ int FeeParam::getROBfromPad(int irow, int icol) const
   //
   // Return on which rob this pad is
   //
-
-  return (irow / mgkNmcmRobInRow) * 2 + getColSide(icol);
+  //LOG(info) << "getROBfromPad  row : "<< irow << "col : " << icol << " answer : " <<  (int)((int)irow / (int)mgkNmcmRobInRow) * 2 + getColSide(icol)<< " with : " << (irow / mgkNmcmRobInRow )*2 << " + " << getColSide(icol);
```

Comment:
This should be removed

Reply:
removed as per comment previous

Reply:
they should be debug bug are commented out for performance reasons, apparently my sed did not find those.